wiki/Difficulties/en.md

```diff
@@ -10,18 +10,20 @@ The difficulty of a beatmap *describes* the level of skill needed to complete it
 
 ![Star rating ranges](SR_range.png "Star rating ranges")
 
-From the [Ranking Criteria](/wiki/Ranking_Criteria), a star rating of:
+For the [Ranking Criteria](/wiki/Ranking_Criteria), star ratings typically correspond to these difficulties:
```

Comment:
Was there a reason to change `From` to `For`?

can probably omit `typically` from the sentence if they are going to be defined that way in https://github.com/ppy/osu-web/pull/4181

Reply:
"For" because the ranking criteria don't actually define the rating brackets (this page does), and "typically" because it's completely acceptable for difficulties to be out of the listed ranges.

Reply:
ok